docker/docker-compose.yml

```diff
@@ -322,3 +322,29 @@ services:
     # See the following:
     # - https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-docker.html,
     # - https://github.com/deviantony/docker-elk/issues/243
+
+  rss-bridge:
+    image: rssbridge/rss-bridge:2022-01-20
+    container_name: 'rss-bridge'
+    # See RSS-Bridge docs for config files:
+    # Whitelist - https://github.com/RSS-Bridge/rss-bridge/wiki/Whitelisting
+    # Configuration - https://github.com/RSS-Bridge/rss-bridge/wiki/Custom-Configuration#timezone
+    volumes:
+      - ../config/rss-bridge:/config
+    depends_on:
+      - traefik
+    labels:
+      # Enable Traefik
+      - 'traefik.enable=true'
+      # Traefik routing for the rss-bridge service at /v1/rss-bridge
+      - 'traefik.http.routers.rss_bridge.rule=PathPrefix(`/${API_VERSION}/rss-bridge`)'
+      # Specify the RSS-Bridge service port
+      - 'traefik.http.services.rss_bridge.loadbalancer.server.port=80'
```

Comment:
I don't think we can use port 80 because it's being used by something else. Let's define the port as an environment variable like {RSS_PORT} and bind it here instead of hardcoding it.

Reply:
The port 80 is the port used inside the container, not in the host, I believe. In the `rss-bridge` docker installation [instructions](https://github.com/RSS-Bridge/rss-bridge/wiki/Docker), it says something along those lines. If I change it to another port number, it gives a `Bad Gateway` response.

Reply:
I think the port 80 in the host is used by `nginx`, by the way.

Reply:
Yes, you have to set the port of the running container here, which in the case of `rss-bridge` is `80`, since it runs Apache.  This is correct.

Reply:
Thanks for the clarification, guys :)

Reply:
I find the Traefik docs 100% crystal clear...once I've spent 4 hours being lost reading them :). It always takes me half-a-day to relearn how it works.

I should probably add a doc to the new doc system...